src/compute/src/logging/compute.rs

```diff
@@ -1176,3 +1196,14 @@ where
 
     sum
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[mz_ore::test]
+    fn test_compute_event_size() {
+        // This could be a static assertion, but we don't use those yet in this crate.
+        assert_eq!(48, std::mem::size_of::<ComputeEvent>())
+    }
+}
```

Comment:
One reason we haven't added such tests is that it might be different on ARM and x86, and it's very confusing to diagnose if this happens.

Reply:
The test passes on my machine (M1 MacBook), so I think it's fine to leave it as it.